ui-tests/src/test/java/org/openmrs/reference/EndVisitTest.java

```diff
@@ -33,7 +33,6 @@ public void setUp() throws Exception {
     }
 
     @Test
-    @Ignore
```

Comment:
you can also remove the package for the @ignore from this file

Reply:
Thanks @HerbertYiga